### PR TITLE
Added Aruba CX switch module

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Aruba CX switch module

# Impacts
Switch module

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries
(REQUIRED, but may be optional...)
## New Features
* Aruba CX support

### CODE REVIEW

File: lib/pf/Switch/Aruba/cx.pm [hunk 1]
```diff
@@ -0,0 +1,219 @@
+package pf::Switch::Aruba::cx;
+
+=head1 NAME
+
+pf::Switch::Aruba::cx
+
+=head1 SYNOPSIS
+
+Module to manage rebanded Aruba HP CX Switche
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/cx.pm]: I think its spelled chwitche

File: lib/pf/Switch/Aruba/cx.pm [hunk 2]
```diff
@@ -0,0 +1,219 @@
+package pf::Switch::Aruba::cx;
+
+=head1 NAME
+
+pf::Switch::Aruba::cx
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/cx.pm]: To follow our naming standards, we should use pf::Switch::Aruba::CX

File: lib/pf/Switch/Aruba/cx.pm [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+
+=head1 SYNOPSIS
+
+Module to manage rebanded Aruba HP CX Switche
+
+=head1 STATUS
+
+=over
+
+=item Supports
```
Comment (julsemaan) [inline lib/pf/Switch/Aruba/cx.pm]: Can you confirm that this switch indeed supports all the features that are outlined there (like dACL, RADIUS CLI login, etc)

Comment (fdurand) [reply lib/pf/Switch/Aruba/cx.pm]: It's based on the Aruba::5400 who support that.

**Pull request merged**